tests/test_file.py

```diff
@@ -414,6 +415,66 @@ def try_get():
                 url, requests.codes.bad_request)
         try_get()
 
+    @testmode.standalone
```

Comment:
This is not a standalone test, since it depends on a step function being deployed.

Consider manually checking out, i.e., manually copying to the expected destination (dss/storage/checkout/file.py:get_dst_key), and then execute the second request if you want this to be a standalone test.

There are certainly some significant advantages to this approach, as you can test the checksum-not-matching code path.